packages/medusa/src/migrations/1659501357661-tax_inclusive_pricing.ts

```diff
@@ -0,0 +1,45 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export const featureFlag = "tax_inclusive_pricing"
```

Comment:
though: `TaxInclusivePricingFeatureFlag.key` just to not have any typo wdyt? at the moment I am the only one to do it that way (which I prefer). So either, I do it you're way or you does mine :) I let you choose

Reply:
I think it's better to reference it as well, lets go with that

Reply:
👌